cmd/sink/main.go

```diff
@@ -28,7 +28,7 @@ func main() {
 	if err != nil {
 		logger.Printf("Could not start tracing: %s\n", err.Error())
 	}
-	httpClient, err := cloudevents.NewHTTP(
+	httpClient, _ := cloudevents.NewHTTP(
```

Comment:
I agree that there is an issue here, but I think the right solution is to add the same kind of `err != nil` check that is on line 38 after the function call here. Not ignoring the returned error entirely. I must have missed this (and everyone that reviewed my PR :sweat_smile: )

Reply:
yes!! I'll address the error as you suggest, just added this line because I was advertise of it in https://github.com/maruiz93/kubearchive/actions/runs/9416549251/job/25939886827

The good news is that with the latest updates on the CI we won't miss this kind of things anymore :)

Reply:
I've undone my change as the error management is already in place